.gitignore

```diff
@@ -8,4 +8,11 @@ riderModule.iml
 metrics.json
 plot.html
 results.bin
-.vscode/
\ No newline at end of file
+.vscode/
+/.vs/road-to-orleans/v16
```

Comment:
Think `**/.vs` will ignore all the vs folders in repo.

Reply:
Good point.